debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/ValueTypeHelper.java

```diff
@@ -112,7 +113,8 @@ private J9ClassPointer findJ9ClassInFlattenedClassCacheWithFieldNameImpl(Structu
 
 				int cacheLength = length.intValue();
 				for (int i = 0; i < cacheLength; i++) {
-					String field = J9UTF8Helper.stringValue(((J9ROMNameAndSignaturePointer)flattenedClassCacheEntry_NaS.invoke(entry)).name());
+					J9UTF8Pointer name = ((J9ROMFieldShapePointer)flattenedClassCacheEntry_field.invoke(entry)).nameAndSignature().name();
```

Comment:
same thing here with name, it is not being used. You can just wrap the `stringValue(` call around